tests/model_serving/model_server/conftest.py

```diff
@@ -359,6 +359,9 @@ def ovms_kserve_inference_service(
     if (scale_target := request.param.get("scale-target")) is not None:
         isvc_kwargs["scale_target"] = scale_target
 
+    if (stop_resume := request.param.get("stop")) is not None:
+        isvc_kwargs["stop_resume"] = stop_resume
```

Comment:
You can specify a default value, and you won't need the `if`:
```suggestion
    isvc_kwargs["stop_resume"] = request.param.get("stop", False)
```